src/play/loader/sample-loader.ts

```diff
@@ -0,0 +1,37 @@
+import { executeSteps, LoadCallback } from "./executor";
+import { Howl } from "howler";
+
+export async function loadSamples(
+  input: Map<string, Blob>,
+  cb: LoadCallback
+): Promise<Map<string, Howl>> {
+  const samples = new Map<string, Howl>();
+
+  await executeSteps(cb, [
+    {
+      weight: 1,
+      async execute(cb) {
+        let decoded = 0;
+        for (const [name, blob] of input.entries()) {
+          cb(
+            decoded / input.size,
+            `Loading Samples (${decoded + 1} / ${input.size})`
+          );
+
+          const howl = new Howl({
+            src: URL.createObjectURL(blob),
+            html5: true,
```

Comment:
Is there any reason in particular that your using html5 audio here?

Reply:
Using HTML5 audio exposes the underlying audio to the operating system, allowing things like pausing with hardware pause keys.

Reply:
No, i don't have any reasons for using it. I just left it as it was in the audio loading.